examples/decoding/plot_ssd_spatial_filters.py

```diff
@@ -2,24 +2,21 @@
 ===========================================================
 Compute Sepctro-Spatial Decomposition (SDD) spatial filters
 ===========================================================
-
 In this example, we will compute spatial filters for retaining
 oscillatory brain activity and down-weighting 1/f background signals
 as proposed by [1]_.
 The idea is to learn spatial filters that separate oscillatory dynamics
 from surrounding non-oscillatory noise based on the covariance in the
 frequency band of interest and the noise covariance absed on surrounding
 frequencies.
-
 References
 ----------
-
 .. [1] Nikulin, V. V., Nolte, G., & Curio, G. (2011). A novel method for
        reliable and fast extraction of neuronal EEG/MEG oscillations on the
        basis of spatio-spectral decomposition. NeuroImage, 55(4), 1528-1535.
 """
-# Author: Denis A. Engemann <denis.engemann@gmail.com>
-#
+# Author: Denis A. Engemann <denis.engemann@gmail.com>, &
+# Victoria Peterson <victoriapeterson09@gmail.com>
```

Comment:
Very good. I was gonna suggest this.

Reply:
I think btw. the typical pattern is not to use & and indent the second line to the first letter of the author in the first line. ... details.